templates/shared_services/firewall/terraform/firewall.tf

```diff
@@ -101,7 +101,10 @@ resource "azurerm_firewall_application_rule_collection" "shared_subnet" {
       "graph.microsoft.com",
       "login.microsoftonline.com",
       "aadcdn.msftauth.net",
-      "graph.windows.net"
+      "graph.windows.net",
```

Comment:
This is beginning to look like a catch all list and we loose the ability to know which domain is required for what.
I think it makes more sense to create a rule specifically for Nexus (targeting the shared subnet as a whole). Please discuss and resolve this comment per the conclusion in the team.
cc: @marrobi

Reply:
Good idea, have amended